generator/routing_index_generator.cpp

```diff
@@ -251,13 +252,55 @@ void CalcCrossMwmTransitions(string const & path, string const & mwmFile, string
       prevPointIn = currPointIn;
     }
   });
-  LOG(LINFO, ("Transitions finished, transitions:", transitions.size(), ", elapsed:",
-              timer.ElapsedSeconds(), "seconds"));
+}
+
+/// \brief Fills |transitions| for transit id case. That means for VehicleType::Transit.
```

Comment:
Could you please paraphrase your comment? It's unclear to me.

Reply:
I write it this way:
`/// \brief Fills |transitions| for transit case. That means Transition::m_roadMask for items in
/// |transitions| will be equal to VehicleType::Transit after call of this method.`
Is it ok with you?